src/query/db_accessor.hpp

```diff
@@ -694,6 +694,8 @@ class SubgraphDbAccessor final {
   std::optional<VertexAccessor> FindVertex(storage::Gid gid, storage::View view);
 
   Graph *getGraph();
+
+  storage::StorageMode GetStorageMode() const noexcept;
```

Comment:
I’ll make it consistent, thanks!

Reply:
Thanks for the suggestion! In the end, I’ve opted not to do it because of consistency:
* all `DbAccessor` method implementations are in the .hpp file already
* all `SubgraphDbAccessor` method implementations are in the .cpp file already